docs/installLiberty.md

```diff
@@ -9,16 +9,16 @@ In certain cases, the Liberty license code may need to be provided in order to i
 | Attribute | Description | Required |
```

Comment:
Are all of these attributes relative to the liberty ` install` block?  We should be clear on that.